SPEC.md

```diff
@@ -314,6 +324,36 @@ String is either a [Quoted String](#quoted-string) or a
 [Raw String](#raw-string). Quoted Strings may include escaped characters, while
 Raw Strings always contain only the literal characters that are present.
 
+Strings _MUST_ be represented as UTF-8 values.
+
+Strings _MUST NOT_ include the code points for [disallowed literal
+code points](#disallowed-literal-code-points) directly. If needed, they can be
+specified with their corresponding `\u{}` escape.
+
+### Multi-line Strings
+
+Strings may span multiple lines with literal Newlines, in which case the
+resulting String is "dedented" according to the line with the fewest number of
+Whitespace characters preceding the first non-Whitespace character. That is,
```

Comment:
Should make it excessively clear that only *literal* whitespace is removed. Escaped whitespace counts as content.